interface/src/app/node/high_range_exp.tpl.html

```diff
@@ -30,15 +30,43 @@
           <!-- Location of vega-lite widget -->
           <div id="{{ 'exp-' + $index }}"></div>
 
-          <ol ng-if="exp.isExpanded">
-            <li ng-if="!exp.sampleStatus"
-                ng-repeat="sample in exp.samples | orderBy: '-activity'">
-              Activity Value: <b>{{sample.activity.toFixed(5)}}</b><br/>
-              Sample Name: <b>{{sample.name}}</b><br/>
-              Data Source: <b>{{sample.ml_data_source}}</b>
-            </li>
+          <div ng-if="exp.isExpanded">
+            <div ng-if="!exp.sampleStatus">
+              <table class="table table-bordered table-condensed table-hover">
+                <thead>
+                  <tr>
+                    <th>#</th>
+                    <th>Activity</th>
+                    <th>Sample Name</th>
+                    <th>Data Source</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr ng-repeat="sample in exp.samples | orderBy: '-activity'"
+                      ng-class="{success: sample.selected}">
```

Comment:
Okay, I see that now. The intended semantics of the Bootstrap class is for things like alerts and such, where `Danger`, `Info`, and `Warning` (described at @dongbohu's link) would also make sense. Using it here is harmless since we already depend upon Bootstrap heavily, but we really ought to be defining our own semantic classes and using those instead. Not a big deal, but worth addressing along with the rest of Issue #76.